slack-java-client/src/main/java/com/hubspot/slack/client/SlackWebClient.java

```diff
@@ -998,11 +1016,13 @@ protected String getInitialOffset() {
     return responseFuture;
   }
 
-  private void acquirePermit(SlackMethod method) {
-    double acquireTime = getSlackRateLimiter().acquire(config.getTokenSupplier().get(), method);
-    if (acquireTime > 5.0) {
-      LOG.warn("Throttling {}, waited {} seconds to acquire permit to run", method, acquireTime);
+  private double acquirePermit(SlackMethod method) {
+    double acquireSeconds = getSlackRateLimiter().acquire(config.getTokenSupplier().get(), method);
+    if (acquireSeconds > 5.0) {
```

Comment:
Can we make this configurable? Or at the very least a constant?

Reply:
I made it a constant. Making it configurable is out of scope and should be in a different PR if people need it